README.md

```diff
@@ -150,6 +150,22 @@ These should be supplied as a valid JSON object, quoted as a string in your GitH
 
 Pipeline config profiles to use. Should be comma separated without spaces.
 
+### `nextflow_config`
+
+**[Optional]** Nextflow config options.
+
+Useful to pass custom Nextflow config options to the `tw launch` command e.g.
+
+```yaml
+jobs:
+  run-tower:
+    steps:
+      - uses: nf-core/tower-action@v2
+        with:
+          nextflow_config: 'process.errorStrategy = 'retry'\nprocess.maxRetries = 3'
```

Comment:
Can you make this a multi line string instead please? With `|` or `>`, whichever preserves new lines (the former I think but I always have to look it up to be sure)

Reply:
Like this? https://github.com/nf-core/tower-action/pull/9/commits/f9285002f6897b7bce989e1446c15627a230912c

Reply:
Much nicer 😊